output/Form.md

```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
 -   [*Benefits*]{.underline}*: What benefits were realised (to date) through
     this initiative, and for whom?*
 
+As a leading international research group and a World Health Organization's
+collaborating centre, we work daily with researchers and professionals living
+abroad. This requires tools that help tracking exchanges between labs in a
+comprehensive way, such as statistical code, manuscript drafts, or meeting
+minutes. By taking this approach, we have not only foster a stronger open
+science practice within our lab, but also across different research consortia in
+the European Union (EU). This may potentially spread through different
+institutes in the region and improve reproducibility rates for the whole
+research community.For example,in the EU-funded project RESPOND, we have already
```

Comment:
Has that article used Github actually? If you want an example that uses it, you can link this repo: https://github.com/CCOMS-UAM/mapping-initiatives

Reply:
Also, mind these sentences dismissed from the "Description" section:

https://github.com/CCOMS-UAM/yos-award/blob/29e6f2eff0a4952bc8a02a103b6d237f2c23602b/output/Form.md?plain=1#L65